scripts/run_ginkgo.sh

```diff
@@ -37,3 +37,7 @@ TEST_SOURCE_ROOT="$TEST_SOURCE_ROOT" ginkgo run -procs=5 tests/precompile \
 ./tests/warp/warp.test \
   --ginkgo.vv \
   --ginkgo.label-filter=${GINKGO_LABEL_FILTER:-""}
+
+ ./tests/precompile/warp/warp.test \
```

Comment:
not sure we need a new directory for these tests? could we add them to the existing warp test instead? or extend the existing warp test to verify the events match the abi?